db/migrate/20220902115804_create_response_items.rb

```diff
@@ -0,0 +1,13 @@
+class CreateResponseItems < ActiveRecord::Migration[7.0]
+  def change
+    create_table :response_items do |t|
+      t.belongs_to :query, index: true, foreign_key: true
+      t.integer :source, null: false
+      t.integer :media_type, null: false
```

Comment:
this one too.

Reply:
About indexing those two fields... Actually I think all filtering operation will be perfromed on result items beloging to specific query. In this case having single column indices by media_type or source probably not good idea as  existing index by query_id should provide better selectivity.

If in future we'll see that user often uses (for example) additional filtering by media_type, we may consider adding *composite* index by query_id and media_type. But adding too much indices also can worsen performance. So I would propose to leave it as-is for now.